docs/data/data-grid/components/toolbar/toolbar.md

```diff
@@ -87,3 +86,11 @@ It implements the roving tabindex pattern.
 |                            <kbd class="key">Right</kbd> | Moves focus to the next button.          |
 |                             <kbd class="key">Home</kbd> | Moves focus to the first button.         |
 |                              <kbd class="key">End</kbd> | Moves focus to the last button.          |
+
+## Legacy Toolbar
+
+:::warning
+Deprecated API.
+:::
+
+The components above replaced the [Legacy Toolbar](/x/react-data-grid/components/#legacy-toolbar), e.g. `<GridToolbarContainer>`.
```

Comment:
Make it cleare to people, they will most likely ask what's the difference with what they are used to.